packages/google_maps_flutter/google_maps_flutter_web/lib/src/google_maps_controller.dart

```diff
@@ -163,6 +185,10 @@ class GoogleMapController {
 
   // Funnels map gmap events into the plugin's stream controller.
   void _attachMapEvents(gmaps.GMap map) {
+    map.onTilesloaded.first.then((event) {
```

Comment:
There's [no `map.onReady` event](https://developers.google.com/maps/documentation/javascript/reference/map#Map-Events), so we synthesize it from the first `onTilesloaded` event 🤷 (_[years of discussion](https://stackoverflow.com/questions/832692/how-can-i-check-whether-google-maps-is-fully-loaded)_)